packages/amplify-app/src/scripts/amplify-push.js

```diff
@@ -24,14 +25,25 @@ function getNamedProfiles() {
   return namedProfiles;
 }
 
-function checkIfProfileExists(profileToUse) {
+async function checkIfProfileExists(profileToUse) {
   const namedProfiles = getNamedProfiles();
-
-  if (!namedProfiles || !namedProfiles[profileToUse]) {
-    console.log(`Profile with name ${profileToUse} not found. Please create one`);
-    return false;
+  if (Object.keys(namedProfiles).length) {
+    if (namedProfiles[profileToUse]) {
+      return profileToUse;
+    } else if (Object.keys(namedProfiles).length === 1) {
+      return Object.keys(namedProfiles)[0];
+    }
```

Comment:
Do we want to still ask the customer if they want to use the only profile available?

Reply:
They are implicitly choosing that profile if they are going through this flow and that is the only profile that exists

Reply:
Changing to ask even if is just one profile